src/proto/macro.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto2";
+
+package mixxx.track.io;
+
+option optimize_for = LITE_RUNTIME;
+
+message Macro {
+  message Action {
+    optional uint32 type = 1;
+    optional uint64 position = 2;
+    optional uint64 target = 3;
```

Comment:
I think we should use names that also indicate the unit (samples/frames/milliseconds?)

Reply:
1. This is a proto
2. Yes, we can use FramePos when the other GSoC project finishes.

Reply:
Yes, I know that this is a proto so we need to use `uint64`. But my point was that it the member should be named appropriately, e.g.:

```suggestion
    optional uint64 sourcePositionMillis = 2;
    optional uint64 targetPositionMillis = 3;
```